source/_components/watson_iot.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: "IBM Watson IoT Platform"
+description: "Record events in the IBM Watson IoT Platform."
+date: 2018-04-03 22:09
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ibm.png
+ha_category: History
+ha_release: 0.68
+---
+
+The `watson_iot` component enables you to link the devices in home-assistant
+with an [IBM Watson IoT Platform instance](https://www.ibm.com/us-en/marketplace/internet-of-things-cloud).
+
+## {% linkable_title Configuration %}
+
+To use this component, you first need to register a gateway device type and then
+a gateway device in your IoT platform instance. For instructions on how to do
+this check the [official documentation](https://console.bluemix.net/docs/services/IoT/gateways/dashboard.html#IoT_connectGateway)
```

Comment:
Please indicate whether it is FREE or paid so that users are not wasting their time.

Reply:
They have a free tier, which is sufficient imho.